src/vm.sh

```diff
@@ -18,6 +20,14 @@ function vm_mount()
   qemu_img_path="${qemu_img_path:-${configurations[qemu_path_image]}}"
   mount_point_path="${mount_point_path:-${configurations[mount_point]}}"
 
+  if [[ ! -r "${prefix}boot/vmlinuz-$(uname -r)" ]]; then
+    say 'To mount the VM, the kernel image needs to be readable'
+    if [[ $(ask_yN 'Do you want to make your host kernel image readable?') =~ 0 ]]; then
+      return 125 # ECANCELED
+    fi
+    cmd_manager "$flag" "sudo dpkg-statoverride --update --add root root 0644 ${prefix}boot/vmlinuz-$(uname -r)"
```

Comment:
This command will only work on Debian-based systems; check if the host system is Debian-based before you use this command. If it is not a Debian system, use the standard chmod.